src/pdf_parser_nutrient.py

```
@@ -1,22 +1,25 @@
-import os
 import shutil
 import subprocess
+from pathlib import Path
+
+_pdf_to_md_bin = shutil.which("pdf-to-markdown")
+if _pdf_to_md_bin is None:
+    raise ImportError(
+        "pdf-to-markdown not found on PATH. "
+        "Install it with: npm install -g @pspdfkit/pdf-to-markdown"
+    )
```

Ruff (0.15.10)

[warning] Avoid specifying long messages outside the exception class

(TRY003)